api/swagger.yml

```
@@ -2816,6 +2816,35 @@ paths:
+          description: Scope of the role to delete
+      responses:
+        200:
+          $ref: "#/components/responses/Ok"
+        400:
+          $ref: "#/components/responses/BadRequest"
+        404:
+          $ref: "#/components/responses/NotFound"
+        500:
+          $ref: "#/components/responses/Internal"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.38.0)

[error] too many spaces inside empty brackets

(brackets)